age calculator.php

```diff
@@ -146,28 +146,28 @@
         $year =$_POST["year"];
         $curent_day = date('day');
         $curent_month = date('month');
-        $curent_year = date('Y');
+        $curent_year = date('Year');
 
```

Comment:
there is not form validation

Reply:
sir i will correct  this and upload it again.